### ISSUE TITLE
Browserify and window.FontFaceOnload

### ISSUE THREAD

**Issue by yoyurec** [zachleat/fontfaceonload#18]
can't use with browserify, b'cose CommonJS modules doesn't work with global scope

**Comment by bevacqua** [comment]
Unobtrusive:

```json
{

  "browser": {
    "fontfaceonload": "./node_modules/fontfaceonload/dist/fontfaceonload.js"
  },
  "browserify-shim": {
    "fontfaceonload": "FontFaceOnload"
  },
  "browserify": {
    "transform": ["browserify-shim"]
  },
  "dependencies": {
    "browserify": "10.1.3",
    "browserify-shim": "3.8.6",
    "fontfaceonload": "0.1.6"
  }
}
```

**Comment by zachleat** [comment]
Merged with #17
